osf/metadata/serializers/datacite/datacite_tree_walker.py

```diff
@@ -388,6 +391,24 @@ def _visit_related(self, parent_el):
                 datacite_relation,
             )
 
+        # Then add verified links to same relatedIdentifiers element
+        osf_item = self.basket.focus.dbmodel
+        from osf.models import AbstractNode
+
+        if isinstance(osf_item, AbstractNode):
+            gv_verified_link_list = osf_item.get_links()
+            for item in gv_verified_link_list:
+                verified_link, resource_type = item.get('target_url', None), item.get('resource_type', None)
+                if verified_link and resource_type:
+                    if verified_link and isinstance(verified_link, str) and smells_like_iri(verified_link):
```

Comment:
`smells_like_iri()` already checks for `str` type, so `if verified_link and smells_like_iri(verified_link)` should be good.